index.html

```diff
@@ -52,45 +52,43 @@ <h2 class="hero__description">Subscribe today and get notified first when we lau
         <div class="container">
             <div class="offer__content">
                 <div class="offer__table">
-                    <div class="offer__collumn">
+                    <div class="offer__column">
                         <div class="offer__pic">
-                            <img src="./src/assets/icon/illustration1.svg" class="offer__img1" alt="ikona procentów">
+                            <img src="./src/assets/icon/illustration1.svg" class="offer__img%" alt="icon of percentage">
                         </div>
                         <div class="offer__inside">
                             <h2 class="offer__header">Title 1</h2>
-                            <img src="./src/assets/icon/icon-underline.svg" class="offer__icon"
-                                alt="ikona podkreslenie">
+                            <img src="./src/assets/icon/icon-underline.svg" class="offer__icon" alt="icon of underline">
                             <p class="offer__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                                 sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
                                 veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
                                 consequat.
                             </p>
                         </div>
                     </div>
-                    <div class="offer__collumn">
+                    <div class="offer__column">
                         <div class="offer__pic">
-                            <img src="./src/assets/icon/illustration2.svg" class="offer__img2" alt="ikona lekarzy">
+                            <img src="./src/assets/icon/illustration2.svg" class="offer__imgDoctors"

```

Comment:
you don't need exact class names on the images, do you?

Reply:
done